style.css

```diff
@@ -0,0 +1,411 @@
+/* stylelint-disable CssSyntaxError */
+/* stylelint-disable no-descending-specificity */
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  overflow-x: hidden;
+}
+
+body {
+  background-color: rgb(248, 248, 248);
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 15px 20px;
+}
+
+header #logo a {
+  font-size: 1.3rem;
+  font-weight: 800;
+  color: #6070ff;
+}
+
+header #menu ul a img {
+  width: 40px;
+  height: 40px;
+}
+
+.section-1 {
+  background: url(image/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 80vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 10px;
+}
+
+.section-1 h2 {
+  font-size: 2.5rem;
+  color: #172b4d;
+  padding: 0 20px;
+  line-height: 1.4;
+}
+
+.section-1 p {
+  font-size: 1.2rem;
+  color: #344563;
+  padding: 0 20px;
+  line-height: 1.5;
+}
+
+.section-1 h3 {
+  font-weight: 100;
+  color: #6070ff;
+  padding: 0 20px;
+}
+
+.section-1 .nav-links ul {
+  display: flex;
+  padding: 0 20px;
+  align-items: center;
+  gap: 15px;
+  height: 50px;
+}
+
+.section-1 .nav-links ul li {
+  width: 30px;
+  height: 30px;
+}
+
+.container {
+  margin-top: 100px;
+  padding: 20px;
+  background-color: rgb(248, 248, 248);
+}
+
+.card {
+  background-color: white;
+  box-shadow: 0 0 1px;
+  border-radius: 15px;
+  padding: 20px;
+  margin-bottom: 100px;
+}
+
+.container .profile {
+  width: 320px;
+  height: 30vh;
+  overflow: hidden;
+}
+
+.container .profile img {
+  width: 100%;
+  height: 100%;
+  border: 10px;
+  border-radius: 15px;
+}
```

Comment:
- [ ] Kindly make sure you stick to the image as much as possible. the spacing, typo are supposed to match the figma design.
`Hint: You may consider making your image responsive by applying the:`
```css
width: 100%,
height: auto
// also you consider setting the
object-fit: container
```
- [ ] Kindly make sure the list element of `backend ...` are well aligned as in the template
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212124706-e6690fb8-576f-4fc2-aef0-85698cd0f5c7.png)

The Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212124864-33ea8e1b-a39c-4e97-9eeb-62dd0d498de8.png)